lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -243,32 +253,32 @@
   openssl_certificate:
     path: /etc/ssl/crt/example.com.crt
     provider: assertonly
-    keyUsage:
+    key_usage:
       - digitalSignature
       - keyEncipherment
-    keyUsage_strict: True
+    key_usage_strict: true
 
 - name: Ensure that the existing certificate can be used for client authentication
   openssl_certificate:
     path: /etc/ssl/crt/example.com.crt
     provider: assertonly
-    extendedKeyUsage:
+    extended_key_usage:
       - clientAuth
 
 - name: Ensure that the existing certificate can only be used for client authentication and time stamping
   openssl_certificate:
     path: /etc/ssl/crt/example.com.crt
     provider: assertonly
-    extendedKeyUsage:
+    extended_key_usage:
       - clientAuth
       - 1.3.6.1.5.5.7.3.8
-    extendedKeyUsage: strict
+    extended_key_usage_strict: true
```

Comment:
IMHO this was wrong before, wasn't it?

Reply:
Seems like a leftover from my version, so yes, it likely was wrong.